developers/bindings/index.md

```diff
@@ -692,6 +692,56 @@ It uses the `DiscoveryResultBuilder` to create the discovery result.
 This list will be given to the constructor of `AbstractDiscoveryService` and can be requested by using `DiscoveryService#getSupportedThingTypes` method.
 
+### Representation Property
+
+The name of one of the properties or configuration parameters, which discriminates the discovery result against a) other results of the same type, or b) already existing manually created Things.
+Typically this is a serial number, IP or MAC address.
+The representation property is used to auto-ignore discovery results of things that already exist in the system.
+This can happen a) if a thing has been created manually, or b) it has been discovered separately by two mechanisms e.g. by mDNS, and by NetBios, or UPnP.
+If the new thing goes online, the auto-ignore service of the inbox checks if the inbox already contains a discovery result of the same type where the value of its representation property is identical to the value of the representation property of the newly added thing.
+If this is the case, the thing in the inbox is automatically set to ignored.
+The representation property must be declared in the [thing-types.xml](thing-xml.md#representation-property)
+
+When comparing the representation property of a discovery result with an existing Thing, the framework checks for a match in its `properties` and also its `configuration parameters`.
+
+If defining a representation property for a bridge, its value does not need to be **globally** unique, but only unique within the context of the bridge, so long as the discovery service calls `.withBridge(bridgeUID)` when building the DiscoveryResult. e.g. if bridge A has child things with representation properties of 1, 2, and 3, and bridge B also has child things with representation properties of 1, 2, and 3, they will not conflict.
+
+```java
+DiscoveryResult result = DiscoveryResultBuilder.create(thingUID)
+  .withProperty("uniqueId", nonUniquePropertyValue)
+  .withBridge(bridgeUID) // bridgeUID plus nonUniquePropertyValue are unique
+  .withRepresentationProperty("uniqueId")
+  .build();
+```
+
+Furthermore, if a thing has two configuration parameters where each individually is not globally unique, but the combination of the two is unique, one can define an extra property that combines the two:
+
+```java
+String propValA = "value-of-non-unique-property-A";
+String propValB = "value-of-non-unique-property-B";
+String uniquePropVal = String.format("%s-%s", propValA, propValB);
+...
+DiscoveryResult hub = DiscoveryResultBuilder.create(thingUID)
+  .withProperty("uniqueId", uniquePropVal)
+  .withRepresentationProperty("uniqueId")
+  .build();
+```
+
+If things are created manually, the property or configuration parameter that would match the auto discovery representation property must be set.
+In the case that a `property` will be used to match the representation property its value must be set in the thing handler's `initialize()` method:
```

Comment:
`In the case where a `property` will be used to match the representation property, its value must be set in the Thing handler's...`

Reply:
Ok. Done.